bin/pipeline.sh

```diff
@@ -67,6 +67,7 @@ pipetask --long-log --log-level="$loglevel" qgraph \
     -c calibrate:astrometry.maxMeanDistanceArcsec=0.025 \
     -c calibrate:requireAstrometry=False \
     -c calibrate:requirePhotoCal=False \
+    -c makeWarp:select.maxPsfTraceDelta=0.2 \
```

Comment:
Using ci_hsc to test rejection logic might work once or twice, but it doesn't have many raws to spare.  Can you at least just drop the threshold down to something that only catches one, rather than two?  And in the long term we should think about how to write integration tests for robustness problems - I suspect it'll need to involve an ever-growing repository of sketchy data.

Reply:
The threshold I chose here does catch only a single detector for deselection into the coadd, providing this message in the log (hmmm, I think I'll update the format to print fewer decimal points!):
```
INFO 2023-01-03T18:00:43.019-08:00 lsst.makeWarp.select (makeWarp:{instrument: 'HSC', skymap: 'discrete/ci_hsc', tract: 0, patch: 69, visit: 903334, ...})(selectImages.py:379) - Removing visit 903334 detector 22 because max-min of model PSF trace values across the unmasked detector pixels is not finite or too large: 0.219810 vs 0.200000 (pixels)
```
It is the pre-existing "force astrometry failures" threshold that impacts two (different) detectors.  I looked into increasing the associated threshold to only catch one. The current setting of `maxMeanDistanceArcsec=0.025` catches two with mean and scatter:
```
0.029 +- 0.019 arcsec
0.028 +- 0.018 arcsec
```
(the next highest has `0.024 +- 0.016`).  I could update the threshold to 0.028, but I'm worried this will be too sensitive since only a slight change in previous steps could push these up/down by 0.001 resulting in some potentially annoyingly frequent updates to the flagged dataIds...but if you think we're getting too tight, I can certainly increase that threshold. (And I totally agree that the procedure for such testing needs some re-thinking going forward...)

Reply:
Ok, let's just leave it all as it is for now, then.